cgrok/grok.go

```diff
@@ -2,8 +2,7 @@ package grok
 
 /*
 #cgo CFLAGS: -I. -std=gnu99
-#cgo windows CFLAGS: -IC:/msys64/mingw64/include
```

Comment:
Correct. This is also awesome because you no longer need to install mingw to this particular path. ![](http://images.idiva.com/media/content/2014/Dec/10_signs_8.gif)